internal/provider/resource_bootstrap_git_test.go

```diff
@@ -233,7 +235,7 @@ func TestAccBootstrapGit_Upgrade(t *testing.T) {
 				),
 			},
 			{
-				Config: bootstrapGitVersion(env, "v2.0.0-rc.3"),
+				Config: bootstrapGitVersion(env, utils.DefaultFluxVersion),
```

Comment:
This piggy-backs on the global variable, ensuring we have one less version string to maintain during releases.